config/rewrites.d/developer.rackspace.com.json

```diff
@@ -33,6 +33,13 @@
             "status": 301
         },
         {
+            "from": "^\\/signup",
```

Comment:
Are there any paths beneath `/signup` that we want to redirect, too? If there aren't it might be good to end with a `\\/?$` to be sure we're not matching too much; if there are, we'll need a capture-and-reuse deal to preserve the rest of the URL through the redirect.